### PR TITLE
tools: Add exclude patterns for viablestrict

### PR DESCRIPTION
**Pull request by seemethere**
Adds the ability to specify patterns that we should skip when doing our evaluation of viable strict.

### CODE REVIEW

**Review by jeanschmidt — APPROVED** [review]

File: .github/actions/update-viablestrict/action.yml [hunk 1]
```diff
@@ -21,6 +21,12 @@ inputs:
       considered stable
     required: true
     type: string
+  exclude-checks:
```
Comment (huydhn) [inline .github/actions/update-viablestrict/action.yml:24]: This should be default to an empty string to safely roll out this change, then we can just use `--exclude-checks "${{ inputs.requires }}"` and update the script logic to handle empty string

**Comment by clee2000** [comment]
isn't the include list a regex? Why not use a regex pattern that excludes the rocm manywheel jobs?

**Comment by seemethere** [comment]
> isn't the include list a regex? Why not use a regex pattern that excludes the rocm manywheel jobs?

This is also true, perhaps this is unnecessary, unsure of how I'd test that it does the negative case though.

**Comment by clee2000** [comment]
> > isn't the include list a regex? Why not use a regex pattern that excludes the rocm manywheel jobs?
> 
> This is also true, perhaps this is unnecessary, unsure of how I'd test that it does the negative case though.

I would just run the script with the new regex pattern and see what it returns
